evap/staff/templates/staff_evaluation_textanswers_quick.html

```diff
@@ -155,6 +176,34 @@ <h5 class="modal-title" id="hotkeys-modal-title">Hotkeys</h5>
                 <div class="lcr-right">
                     {% if user.is_manager %}
                         <div data-action-set="reviewing">
+                            <div class="btn-group icon-buttons" role="group">
+                                <!-- Note: The `autocomplete` is not standard compliant, but in practice disables caching -->
```

Comment:
What is the problem case here? The browser checking both checkboxes?

Why do we have two checkboxes? You probably have a good reason, I'd guess we want to have that documented in comment or so.

Reply:
I had it for general sanity. This is all firefox things, they are caching the state (not spec compliant) but have this `autocomplete` to disable it (not spec compliant). Other browsers should just ignore the attribute I guess.

The horror scenario in my head is for quick view where the input is shared across answers. The state _should_ be managed by our javascript, but if anything is ever wrong, I don't want to think about mozilla's shenanigans that day

Reply:
see changes